sklearn/tree/tests/test_export.py

```diff
@@ -207,6 +208,20 @@ def test_graphviz_toy():
 
     assert_equal(contents1, contents2)
 
+    # Test classifier with degraded learning set
+    clf = DecisionTreeClassifier(max_depth=3)
+    clf.fit(X, y_degraded)
+
+    out = StringIO()
+    export_graphviz(clf, out_file=out, filled=True)
+    contents1 = out.getvalue()
+    contents2 = 'digraph Tree {\n' \
```

Comment:
Note from [PEP8](https://www.python.org/dev/peps/pep-0008/):
> The preferred way of wrapping long lines is by using Python's implied line continuation inside parentheses, brackets and braces. Long lines can be broken over multiple lines by wrapping expressions in parentheses. These should be used in preference to using a backslash for line continuation.

For long strings you can do:

```python
contents2 = ('digraph Tree {\n'
             'node ...\n'
             '0 [label ...]\n'
             '}')
```

Alternatively in your particular case, this would work:
```python
contents2 = '\n'.join(['digraph Tree{', 'node ...', '0 [label ...', '...'])
```